src/App.css

```diff
@@ -1,15 +1,164 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: content-box;
+}
+
 header {
   display: flex;
-  justify-content: space-around;
+  justify-content: space-between;
+  align-items: center;
+  height: 5.438rem;
+  border-bottom: solid 1px #e8e8e8;
+}
+
+#inputs {
+  display: grid;
+  grid-template-columns: 5.5fr 2fr 1.5fr;
+  column-gap: 15px;
+  height: 30px;
+}
+
+h1 {
+  height: 2.313rem;
+  margin: 0.375rem 0 0.125rem 0;
+  font-family: Montserrat, Arial, Verdana, Tahoma, sans-serif;
+  font-size: 1.875rem;
+  font-weight: 900;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  margin-left: 6.188rem;
+  gap: 20px;
+}
+
+#userImage {
+  height: 3.188rem;
+  margin-right: 6.188rem;
 }
 
 a {
+  list-style: none;
+  font-family: Montserrat, Arial, Verdana, Tahoma, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 0.7;
   margin: 20px;
   text-decoration: none;
 }
 
-#inputs {
-  display: grid;
-  grid-template-columns: 7fr 3fr;
-  column-gap: 10px;
+h4 {
+  margin: 0 4rem 7px 0;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Verdana, Tahoma, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book {
+  height: 5.8rem;
+  background-color: #fff;
+  border-radius: 4px;
+  border: solid 2px #e8e8e8;
+  padding: 2rem 4rem 1.625rem 1.688rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.booksContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 17px;
+  background-color: #fafafa;
+  min-height: 100vh;
+  padding: 2.7rem 6.25rem 16.563rem 6.25rem;
+}
+
+.removeButton {
+  background: none;
+  border: none;
+  padding: 0;
+  outline: inherit;
+  border-left: solid 2px #e8e8e8;
+  border-right: solid 2px #e8e8e8;
+}
```

Comment:
- [ ] Please kindly ensure your design matches the original design guidelines(font type, font weight, spacing, padding etc)